File: 1007/Go/main.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c, d, result int
	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)
	fmt.Scan(&d)
	result = (a * b) - (c * d)
	fmt.Printf("DIFERENCA = %d\n", result)
}

File: 1007/C++/main.cpp
// https://www.urionlinejudge.com.br/judge/en/problems/view/1007
#include <iostream>

int main() {
  int a, b, c, d, result;
  std::cin >> a >> b >> c >> d;
  result = a * b - c * d;
  std::cout << "DIFERENCA = " << result << std::endl;
  return 0;
}
